### PR TITLE
RHDEVDOCS-4344 - Legacy syslog / fluentd configmaps.

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.8

Issue: [RHDEVDOCS-4344](https://issues.redhat.com//browse/RHDEVDOCS-4344) 

Link to docs preview: https://drive.google.com/file/d/19MU8dAuPp-CEkMkrK8Y9zUJmG13G3eeC/view?usp=share_link 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: logging/cluster-logging-external.adoc [hunk 1]
```diff
@@ -14,16 +14,6 @@ To send logs to other log aggregators, you use the {product-title} Cluster Log F
 ====
 To send audit logs to the default internal Elasticsearch log store, use the Cluster Log Forwarder as described in xref:../logging/config/cluster-logging-log-store.adoc#cluster-logging-elasticsearch-audit_cluster-logging-store[Forward audit logs to the log store].
 ====
-
-When you forward logs externally, the Red Hat OpenShift Logging Operator creates or modifies a Fluentd config map to send logs using your desired protocols. You are responsible for configuring the protocol on the external log aggregator.
```
Comment (kabirbhartiRH) [inline logging/cluster-logging-external.adoc:18]: Could you double check if this line needs to be removed?

Comment (libander) [reply logging/cluster-logging-external.adoc:18]: @alanconway / @periklis / @jcantrill ^ - Thoughts?

**Comment by libander** [comment]
@alanconway - /lgtm please.

**Pull request merged**